src/clj/ataru/organization_service/organization_selection.clj

```diff
@@ -0,0 +1,37 @@
+(ns ataru.organization-service.organization-selection
+  (:require [clojure.string :as string]
+            [ataru.organization-service.organization-service :as organization-service]
+            [medley.core :refer [find-first]]))
+
+(defn- all-organizations
+  [organization-service]
+  (organization-service/get-all-organizations organization-service [{:oid ""}]))
+
+(defn query-organization
+  [organization-service session query]
+  (let [superuser?    (-> session :identity :superuser (boolean))
+        organizations (sort-by
+                        #(some (fn [lang] (-> % :name lang)) [:fi :sv :en])
+                        (if superuser?
+                          (all-organizations organization-service)
+                          (-> session :identity :organizations (vals))))]
+    (take 11
+          (if (or (string/blank? query)
+                  (< (count query) 2))
```

Comment:
`count` pitäis riittää tässä?

Reply:
Toi `blank?` on lähinnä nil-guardina, mut tosiaan sit `nil?` varmaan ajais saman asian...